### ISSUE TITLE
[Bug]: Crashing shortly after running.

### ISSUE THREAD

**Issue by CorneliousJD** [advplyr/audiobookshelf#1963]
### Describe the issue

After recent 2.3.x updates where I had to delete the old db file and let it re-create itself, I have had crashing issues ever since.

Podcasts downloading will seemingly crash, but it's hard to tell if that's what's causing it or not, nothing is actually logged in the logs, the docker container will just crash after running for a while, sometimes download queues are the last thing in the logs, othertimes not.

Very hard to track down what's going on here.

### Steps to reproduce the issue

1. Start container.
2. Wait. 
3. Check back later and see it not running.

### Audiobookshelf version

2.3.3

### How are you running audiobookshelf?

Docker

**Comment by advplyr** [comment]
I've been working on fixing a bug when auto-downloading to a podcast with a lot of episodes downloaded already (like 1000+). Do you have a podcast with many episodes downloaded?
You can also try spacing out the schedule of your podcast episode checks

**Comment by CorneliousJD** [comment]
> I've been working on fixing a bug when auto-downloading to a podcast with a lot of episodes downloaded already (like 1000+). Do you have a podcast with many episodes downloaded? You can also try spacing out the schedule of your podcast episode checks

yes, I have the "stuff you should know" podcast configured and they have  a large number of EP's that I already have downloaded. (over 2,000)

I've set it to change the check from every hour to every 12 hours.
I assume it will crash every 12 hours now instead though and not download anything for that podcast?

**Comment by advplyr** [comment]
What I meant to say is you can set the schedule to download on the 15 minutes mark instead of on the hour if you have other podcasts scheduled to on the hour.

**Comment by CTI-Corey** [comment]
> What I meant to say is you can set the schedule to download on the 15 minutes mark instead of on the hour if you have other podcasts scheduled to on the hour.

Ah, well even after setting that to 12 hours it still crashed within the last 47 minutes of me doing that after starting it fresh. logs only show 

```
2023-07-25 15:40:36] INFO: [Database] Db data loaded in 2.15s
[2023-07-25 15:40:36] INFO: [BackupManager] 7 Backups Found
[2023-07-25 15:40:36] INFO: [LogManager] Init current daily log filename: 2023-07-25.txt
[2023-07-25 15:40:36] INFO: [Watcher] Initializing watcher for "Audio Books".
[2023-07-25 15:40:36] INFO: [Watcher] Initializing watcher for "Podcasts".
[2023-07-25 15:40:36] INFO: Listening on port :80
[2023-07-25 15:40:36] INFO: [Watcher] "Audio Books" Ready
[2023-07-25 15:40:37] INFO: [Watcher] "Podcasts" Ready
[2023-07-25 15:40:55] INFO: [Server] Socket Connected IHZ3HFjjXOLyIuHFAAAB
```

**Comment by CorneliousJD** [comment]
Sorry, replied with wrong github account there. 
but yes it still seems to crash regardless when downloading files - a manual trigger of downloading them crashes the app entirely. 

log FILE shows the following - same thing pretty much though. 
```
{"timestamp":"2023-07-25 16:30:54","source":"Logger.js:114","message":"[Database] Db data loaded in 2.08s","levelName":"INFO","level":2}
{"timestamp":"2023-07-25 16:30:54","source":"Logger.js:120","message":"[BackupManager] Old unsupported backup was found \"2023-07-11T0130.audiobookshelf\" (pre sqlite migration)","levelName":"WARN","level":3}
{"timestamp":"2023-07-25 16:30:54","source":"Logger.js:120","message":"[BackupManager] Old unsupported backup was found \"2023-07-12T0130.audiobookshelf\" (pre sqlite migration)","levelName":"WARN","level":3}
{"timestamp":"2023-07-25 16:30:54","source":"Logger.js:120","message":"[BackupManager] Old unsupported backup was found \"2023-07-13T0130.audiobookshelf\" (pre sqlite migration)","levelName":"WARN","level":3}
{"timestamp":"2023-07-25 16:30:54","source":"Logger.js:120","message":"[BackupManager] Old unsupported backup was found \"2023-07-14T0130.audiobookshelf\" (pre sqlite migration)","levelName":"WARN","level":3}
{"timestamp":"2023-07-25 16:30:54","source":"Logger.js:120","message":"[BackupManager] Old unsupported backup was found \"2023-07-15T0130.audiobookshelf\" (pre sqlite migration)","levelName":"WARN","level":3}
{"timestamp":"2023-07-25 16:30:54","source":"Logger.js:120","message":"[BackupManager] Old unsupported backup was found \"2023-07-16T0130.audiobookshelf\" (pre sqlite migration)","levelName":"WARN","level":3}
{"timestamp":"2023-07-25 16:30:54","source":"Logger.js:114","message":"[BackupManager] 7 Backups Found","levelName":"INFO","level":2}
{"timestamp":"2023-07-25 16:30:54","source":"Logger.js:114","message":"[LogManager] Init current daily log filename: 2023-07-25.txt","levelName":"INFO","level":2}
{"timestamp":"2023-07-25 16:30:54","source":"Logger.js:114","message":"[Watcher] Initializing watcher for \"Audio Books\".","levelName":"INFO","level":2}
{"timestamp":"2023-07-25 16:30:54","source":"Logger.js:114","message":"[Watcher] Initializing watcher for \"Podcasts\".","levelName":"INFO","level":2}
{"timestamp":"2023-07-25 16:30:54","source":"Logger.js:114","message":"Listening on port :80","levelName":"INFO","level":2}
{"timestamp":"2023-07-25 16:30:54","source":"Logger.js:114","message":"[Watcher] \"Audio Books\" Ready","levelName":"INFO","level":2}
{"timestamp":"2023-07-25 16:30:55","source":"Logger.js:114","message":"[Watcher] \"Podcasts\" Ready","levelName":"INFO","level":2}
{"timestamp":"2023-07-25 16:31:10","source":"Logger.js:114","message":"[Server] Socket Connected GlfVOdefJ6-eYuoBAAAB","levelName":"INFO","level":2}
```

**Comment by CorneliousJD** [comment]
No matter what I do or try this container is still crashing now regularly after running for a short period.
Before this it was stable for months on end, it only happened with the upgrade to to 2.3.x

**Comment by advplyr** [comment]
Many issues relating to server crashes have been resolved on the latest release v2.4.2. Let me know if you are still having an issue and we can re-open this.

**Comment by CorneliousJD** [comment]
> Many issues relating to server crashes have been resolved on the latest release v2.4.2. Let me know if you are still having an issue and we can re-open this.

I am unfortunately still seeing a crash after letting it run for a short period.

**Comment by advplyr** [comment]
I just re-read your messages and see that you have a podcast with a large number of episodes. There is a known memory issue with loading podcasts with thousands of episodes that is being tracked in #2075
You would only see this error if you are looking directly at the logs from the container and not from Abs.
I'm pretty sure your issue is the same as #2075 though
